src/header/value.rs

```diff
@@ -289,10 +289,33 @@ impl AsRef<[u8]> for HeaderValue {
 
 impl fmt::Debug for HeaderValue {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        f.debug_struct("HeaderValue")
-            .field("value", &EscapeBytes(self.as_ref()))
-            .field("is_sensitive", &self.is_sensitive)
-            .finish()
+        if self.is_sensitive {
+            f.write_str("Sensitive")
```

Comment:
I wonder if we could maybe do something like:

```rust
f.debug_struct("HeaderValue")
    .field("value", &"...")
    .field("is_sensitive", is_sensitive)
    .finish()
```

just to make diffs between debugs of different `HeaderValue` instances a little more consistent perhaps?